src/Servers/IIS/Directory.Build.props

```diff
@@ -0,0 +1,6 @@
+<Project>
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory)..\, Directory.Build.props))\Directory.Build.props" />
+  <PropertyGroup>
+    <KestrelSharedSourceRoot>$(MSBuildThisFileDirectory)..\Kestrel\shared\</KestrelSharedSourceRoot>
```

Comment:
Should we consider moving this to https://github.com/dotnet/aspnetcore/tree/main/src/Shared/Kestrel or something?  Is there a reason it needs to be special?

Reply:
It's grown very organically, it used to only be shared within Kestrel projects, now it's shared across all the servers. It might be worth moving, but not in this PR. Will, file an issue?

Reply:
https://github.com/dotnet/aspnetcore/issues/36035